### DISCUSSION TITLE
how do I close open / close dropdown programmatically through ref ?

### DISCUSSION THREAD

**Discussion by handhikadj** [sagalbot/vue-select#1662] [Help Wanted]
currently can't do this

```vue
<v-select ref="myDropdown" />

methods: {
  myMethod() {
    this.$refs.myDropdown.open()
  }
}
```

**Comment by enzolupia** [comment]
@handhikadj facing the same issue, did you find any solution?

**Reply by handhikadj** [reply]
I think I found the workaround back in the day. I forgot what the solution was. sorry.
will be back if I find the code

**Reply by howard-tzw** [reply]
@handhikadj thanks, need your help!
